std/README.md

```diff
@@ -21,15 +21,18 @@ Don't link to / import any module whose path:
 - Is that of a test module or test data: `test.ts`, `foo_test.ts`,
   `testdata/bar.txt`.
 
-No stability is guaranteed for these files.
+Don't import any symbol with an underscore prefix: `export function _baz() {}`.
+
+These elements are not considered part of the public API, thus no stability is
+guaranteed for them.
```

Comment:
Is this a random inclusion or is there an underscore export in this patch?

Reply:
`_format()`  in `std/testing/asserts.ts`.